cloudpathlib/cloudpath.py

```diff
@@ -453,6 +453,15 @@ def replace(self, target: "CloudPath") -> "CloudPath":
                 f"The target based to rename must be an instantiated class of type: {type(self)}"
             )
 
+        if self.is_dir():
+            raise CloudPathIsADirectoryError(
+                f"Path {self} is a directory; rename/replace the files recursively."
+            )
```

Comment:
Does pathlib error if you call `replace` on a directory? I assume not. If not, can we add an issue to track implementing `replace` for directories?

Reply:
Actually I just checked and I think this is already logged as issue #64